extensions/ql-vscode/test/vscode-tests/jest.activated-extension.setup.ts

```diff
@@ -19,13 +19,6 @@ if (process.env.CI) {
 let removeStorage: tmp.DirResult["removeCallback"] | undefined;
 
 export async function beforeAllAction() {
-  // Set the CLI version here before activation to ensure we don't accidentally try to download a cli
-  await testConfigBeforeEachAction();
```

Comment:
I would have expected this line to still be necessary for setting up mock configs. I think the reason that it's still working is because it's also called in `beforeEachAction`.

Do we also need to remove `CUSTOM_CODEQL_PATH_SETTING.updateValue` from `beforeEachAction`?

Reply:
I've removed some more test setup (https://github.com/github/vscode-codeql/pull/3118/commits/b122caab6ec8733f760f0a006c8837c0ce1d1afe) and things seem to still be working 🎉